dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -16,77 +16,79 @@
 
 from typing import Tuple
 from xlml.apis import gcp_config, metric_config, task, test_config
+import dags.vm_resource as resource
 from dags import gcs_bucket, test_owner
-from dags.vm_resource import TpuVersion, Project, RuntimeVersion
 
 
 # TODO(ranran or PyTroch/XLA team): this is an example for benchmark test with hardcode compatible versions,
```

Comment:
Could you update this TODO to you, as you are the test owner now?

Reply:
Haha, sure. I think we can remove the TODO already.